packages/api/src/check/Checker.ts

```diff
@@ -0,0 +1,9 @@
+import { Mutant } from '../../core';
+
+import { CheckResult } from './CheckResult';
+
+export interface Checker {
+  initialize(): Promise<void>;
```

Comment:
Maybe rename this to `init()`, to be in sync with other apis (i.e. `TestRunner`)
```suggestion
  init(): Promise<void>;
```